app/controllers/ember-community-survey-2020.js

```diff
@@ -337,32 +338,16 @@ const SS_Q305 = {
 // Q3.06: Does your business require applications to be internationalized?
 //------------------------------------------------
 
-const SS_Q306 = {
-  options: {
-    chart: {
-      type: 'pie',
-      colors: [emberColors.blue, emberColors.ember],
-    },
-    title: {
-      text: 'Do you internationalize your applications?',
-    },
-    subtitle: { text: '' },
-    tooltip: { pointFormat: '{point.y:.2f}%' },
-    xAxis: { type: 'category' },
-    yAxis: { title: { text: 'Percent of Responses' } },
```

Comment:
I think these may be ignored in pie charts. I wasn't able to find the text `Percent of Responses` in the HTML, for example.